scripts/integer-tests.sh

```diff
@@ -24,35 +24,64 @@ else
     n_threads=6
 fi
 
-# block pbs are too slow for high params
-# mul_crt_4_4 is extremely flaky (~80% failure)
-# test_wopbs_bivariate_crt_wopbs_param_message generate tables that are too big at the moment
-# same for test_wopbs_bivariate_radix_wopbs_param_message_4_carry_4
-# test_integer_smart_mul_param_message_4_carry_4 is too slow
-filter_expression=''\
+if [[ "${BIG_TESTS_INSTANCE}" == FALSE ]]; then
+    # block pbs are too slow for high params
+    # mul_crt_4_4 is extremely flaky (~80% failure)
+    # test_wopbs_bivariate_crt_wopbs_param_message generate tables that are too big at the moment
+    # same for test_wopbs_bivariate_radix_wopbs_param_message_4_carry_4
+    # test_integer_smart_mul_param_message_4_carry_4 is too slow
+    filter_expression=''\
 'test(/^integer::.*$/)'\
 'and not test(/.*_block_pbs(_base)?_param_message_[34]_carry_[34]$/)'\
 'and not test(~mul_crt_param_message_4_carry_4)'\
 'and not test(/.*test_wopbs_bivariate_crt_wopbs_param_message_[34]_carry_[34]$/)'\
 'and not test(/.*test_wopbs_bivariate_radix_wopbs_param_message_4_carry_4$/)'\
 'and not test(/.*test_integer_smart_mul_param_message_4_carry_4$/)'
 
-export RUSTFLAGS="-C target-cpu=native"
```

Comment:
No more RUSTFLAGS is there a reason behind it ?

Reply:
If we put the RUSTFLAGS here the Makefile may not be able to choose the RUSTFLAGS for the script, I think initially I was thinking about being able to run the script as a standalone, now that most things go through the Makefile I don't think this makes sense anymore

Reply:
Thanks for the explanation.